docs/meetings/dev/2024-10-08

```diff
@@ -0,0 +1,9 @@
+- We talked about how to coordinate interviews with new volunteers and how to schedule calls with them.
```

Comment:
Could you paste the following to the top of the page:

```
# Dev meeting

When: 8/10/24, 20:00 UTC.
Present: Aapeli, Darren, David, Ivan, Jesse, Miuriel, Nicole, Yannic

# Notes
```

Reply:
sure!